src/block/Hopper.php

```diff
@@ -38,7 +38,7 @@ class Hopper extends Transparent{
 	use PoweredByRedstoneTrait;
 
 	/** @var int */
-	private $facing = Facing::DOWN;
+	protected $facing = Facing::DOWN;
```

Comment:
Why do you change it to `protected`?

Reply:
Because in all such cases the variable is protected
Or is there a special reason for the private of this variable in the hopper?

Reply:
Is there a special reason to be protected?

Reply:
Unless there is a reason for increasing the visibility, the visibility should not be increased.